include/basic_key.h

```diff
@@ -234,7 +234,7 @@ namespace pgp {
                 std::array<uint8_t, 20> print{fingerprint()};
 
                 // copy the last 8 bytes into the result container
-                std::array<uint8_t, 8>  result;
+                std::array<uint8_t, 8>  result{};
```

Comment:
I don't think we should do this. It doesn't solve anything, except silencing the check, which isn't worth the additional runtime cost.

Reply:
Reverted